hydradx/tests/test_omnipool_state.py

```diff
@@ -311,8 +311,6 @@ def test_cash_out_accuracy(omnipool: oamm.OmnipoolState, share_price_ratio, lp_i
             )
             lrna_profits[tkn] = withdraw_agent.holdings[tkn] - agent_holdings
 
-        er = 1
```

Comment:
Why is this change in this PR? We have some error flag only used in the trade strategies we are deleting or something?